scripts/update_source_data_schema_changelog.py

```diff
@@ -0,0 +1,198 @@
+import json
+import os
+import sys
+from datetime import datetime
+
+OLD_SCHEMAS_DIR = os.environ.get("OLD_SCHEMAS_DIR")
+NEW_SCHEMAS_DIR = "schemas"
+CHANGELOG_FILEPATH = "changelog/source_data.md"
+
+
+def read_json_file(filepath: str) -> {}:
+    with open(filepath, "r") as rfp:
+        try:
+            return json.load(rfp)
+        except:
+            sys.exit(f"Not a valid JSON at filepath {filepath}.")
+
+
+def read_file(filepath: str) -> str:
+    with open(filepath, "r") as rfp:
+        return rfp.read()
+
+
+def write_file(filepath: str, content: str, mode="a") -> None:
+    with open(filepath, mode) as wfp:
+        wfp.write(content)
+
+
+def sort_schema_changes(schema_changes: {}) -> {}:
+    sorted_data = {}
+
+    for table_name, op_types in sorted(schema_changes.items()):
+        sorted_operations = {
+            op_type: sorted(columns) for op_type, columns in sorted(op_types.items())
+        }
+        sorted_data[table_name] = sorted_operations
```

Comment:
```suggestion
        sorted_data[table_name] = {
            op_type: sorted(columns) for op_type, columns in sorted(op_types.items())
        }
```

nit: can directly assign instead of using a new variable